src/content/guidelines/iconography/library.mdx

```diff
@@ -6,9 +6,8 @@ tabs: ['Library', 'Usage', 'Contribute']
 
 import IconLibrary from '../../../../src/components/IconLibrary';
 
-
 <Row className="tile--group">
-<Column offsetLg="4" colMd="4" colLg="4" noGutterSm>
+<Column offsetLg="4" colSm="3" colMd="4" colLg="4" noGutterSm>
```

Comment:
```suggestion
<Column offsetLg="4" colSm="4" colMd="4" colLg="3" noGutterSm>
```

I believe this is the correct sizing for the cards, however, definitely want to check w/ design. Full width at mobile, 1/2 at tablet, and then they want smaller than normal at the larger breakpoint, so only 3 cols instead of the normal 4.

Reply:
thanks alison! i experimented with different settings to try to find the best behavior. i'll look to your previous work as an example to follow!

Reply:
also hmm in #1485 a specific request is "see the icons when I arrive on the icon page so that I know I'm in the right spot.  (smaller resource cards)", so i'll check with tyler : )

Reply:
there was a lot of unsolved design discussion, so for now to match the IDL site, all resource links have been moved to usage page